### PR TITLE
Fonts icon

### PR DESCRIPTION
**Pull request by k-atwhite**
# Pull Request Template
## Description

Added favicon
Added png
Updated h1 fonts

## Type of change
- [ ] Bug fix 
- [x] New feature
- [ ] Refactoring

## Checklist:
- [x] My code follows the Turing's guidelines of this project
- [x] My changes generate no new errors/warnings

### CODE REVIEW

**Comment by sydnerd** [comment]
I love this font!!!

File: src/components/App/App.js [hunk 1]
```diff
@@ -31,7 +32,10 @@ const App = () => {
     <ThemeContext.Provider value={{ theme, setTheme }}>
       <div className={`App ${theme}`}>
         <header className="App-header">
-          <p>Backyard Burbs</p>
+          <h1>
```
Comment (LKessell) [inline src/components/App/App.js:35]: The h1 is a much better choice here

File: src/components/NavBar/NavBar.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+nav {
+  font-family: "Chelsea Market", cursive;
```
Comment (LKessell) [inline src/components/NavBar/NavBar.css:2]: This font is SO cute!

**Review by LKessell — APPROVED** [review]
Looks great!

**Pull request merged**